IPython/core/tests/test_magic.py

```diff
@@ -623,9 +623,13 @@ def test_extension():
 @skipIf(dec.module_not_available('IPython.nbformat'), 'nbformat not importable')
 class NotebookExportMagicTests(TestCase):
     def test_notebook_export_json(self):
+        _ip = get_ipython()
+        _ip.history_manager.reset()   # Clear any existing history.
+        cmds = [u"a=1", u"def b():\n  return a**2", u"print(a, b())"]
```

Comment:
Can you stick a non-ascii character in one of these as well, to check that it's not broken with unicode.